packages/at_client_flutter/lib/src/services/auth_service.dart

```diff
@@ -36,11 +36,20 @@ class AuthService {
   /// Authenticate an atSign with secondary server
   ///
   ///   [atAuthRequest] - AtAuthRequest containing atSign, AtRootDomain, AtKeysIo.
+	///
+	///		[backupOnKeychain] -  
+	///
   /// Returns [AtAuthResponse] which contains keys and status of authentication
-  Future<AtAuthResponse> authenticate(AtAuthRequest atAuthRequest) async {
+  Future<AtAuthResponse> authenticate(AtAuthRequest atAuthRequest, {bool backupOnKeychain = false}) async {
```

Comment:
I would consider avoiding the idea of introducing flutter specific things... that is what got us here in the first place.

That begs the question of how to make it generic. One possible answer is to introduce an optional `List<AtKeysIo>` to the dart signature which is for additional places to backup the keys during activation.

The idea behind this comment is that this signature stays the same as at_auth's.